eucalyptus.te

```diff
@@ -318,6 +323,7 @@ read_files_pattern(eucalyptus_node_t, dhcpd_state_t, dhcp_etc_t)  # Edge NC gate
 
 read_files_pattern(eucalyptus_node_t, eucalyptus_eucanetd_t, eucalyptus_eucanetd_t)  # https://eucalyptus.atlassian.net/browse/EUCA-12047
 
+write_files_pattern(eucalyptus_cluster_t, eucalyptus_var_run_t, eucalyptus_var_run_t)
```

Comment:
If this is actually meant as a policy for eucalyptus_cluster_t it should be in that section, not eucalyptus_node_t's.  That section contains `manage_files_pattern(eucalyptus_cluster_t, eucalyptus_var_run_t, eucalyptus_var_run_t)`, though, which is a superset of this `write_files_pattern`.  I'm somewhat confused about how this manages to fix anything.